jekyll.gemspec

```diff
@@ -36,11 +36,15 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("i18n",                  "~> 0.7")
   s.add_runtime_dependency("jekyll-sass-converter", "~> 1.0")
   s.add_runtime_dependency("jekyll-watch",          "~> 2.0")
-  s.add_runtime_dependency("kramdown",              "~> 1.14")
   s.add_runtime_dependency("liquid",                "~> 4.0")
   s.add_runtime_dependency("mercenary",             "~> 0.3.3")
   s.add_runtime_dependency("pathutil",              "~> 0.9")
   rouge_versions = ENV["ROUGE_VERSION"] ? ["~> #{ENV["ROUGE_VERSION"]}"] : [">= 1.7", "< 4"]
   s.add_runtime_dependency("rouge",                 *rouge_versions)
   s.add_runtime_dependency("safe_yaml",             "~> 1.0")
+
+  # Depend on kramdown. For kramdown v2, extra gems are required.
+  # https://kramdown.gettalong.org/news.html#kramdown-200-released
+  kramdown_versions = ENV["KRAMDOWN_VERSION"] ? ["~> #{ENV["KRAMDOWN_VERSION"]}"] : [">= 1.17", "< 3"]
+  s.add_runtime_dependency("kramdown",              *kramdown_versions)
```

Comment:
Not sure why RuboCop did not complain about this.. (*all* gems not being in the alphabetic order)

*No complains, no issues.*

Ignoring this..

Reply:
They like blocks of gems to be alphabetical, but since there's a blank line in between, this one is considered separate.